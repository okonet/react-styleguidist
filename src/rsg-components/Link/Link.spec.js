import React from 'react';
import Link, { LinkRenderer } from './LinkRenderer';

const href = '/foo';
const children = 'Foo';

it('renderer should render link', () => {
	const actual = shallow(
		<LinkRenderer href={href} classes={{}}>{children}</LinkRenderer>
	);

	expect(actual).toMatchSnapshot();
});

it('should compose passed class names', () => {
	const actual = render(
		<Link
			href={href}
			className="customClassName"
		>
			{children}
		</Link>
	);

	expect(actual.find('a').hasClass('customClassName')).toEqual(true);
});
