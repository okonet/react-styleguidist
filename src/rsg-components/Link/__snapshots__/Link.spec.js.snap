exports[`test renderer should render link 1`] = `
<a
  className=""
  href="/foo">
  Foo
</a>
`;
